Mathlib/Analysis/LocallyConvex/Basic.lean

```diff
@@ -183,6 +182,18 @@ theorem absorbent_iff_eventually_nhdsWithin_zero :
 
 alias ⟨Absorbent.eventually_nhdsWithin_zero, _⟩ := absorbent_iff_eventually_nhdsWithin_zero
 
+theorem absorbs_iff_eventually_nhds_zero (h₀ : 0 ∈ s) :
+    Absorbs 𝕜 s t ↔ ∀ᶠ c : 𝕜 in 𝓝 0, MapsTo (c • ·) t s :=
+  ⟨fun h ↦ h.eventually_nhds_zero h₀, fun h ↦ absorbs_iff_eventually_nhdsWithin_zero.2 <|
+    h.filter_mono inf_le_left⟩
```

Comment:
Could you instead replace `Absorbs.eventually_nhds_zero` by
```lean
theorem absorbs_iff_eventually_nhds_zero (h₀ : 0 ∈ s) :
    Absorbs 𝕜 s t ↔ ∀ᶠ c : 𝕜 in 𝓝 0, MapsTo (c • ·) t s := by
  rw [absorbs_iff_eventually_nhdsWithin_zero, ← nhdsWithin_compl_singleton_sup_pure, 
      Filter.eventually_sup, Filter.eventually_pure, and_iff_left]
  intro x _
  simpa only [zero_smul]
```
and add an alias for the forward direction?

Reply:
I decided not to use `alias` to get `Absorbs` argument first. I think that it's more natural for dot notation lemmas.